src/main/resources/config/liquibase/changelog/20210929120000_add_date_of_birth.xml

```diff
@@ -21,7 +21,7 @@
     -->
     <changeSet id="20210929120000-1" author="alexey@thehyve.nl">
         <addColumn tableName="subject">
-            <column name="date_of_birth" type="date"/>
+            <column name="date_of_birth" type="date" defaultValueDate="1999-01-01"/>
```

Comment:
No changes should be made to existing Liquibase changeSets, because it leads to undefined state of the database. If you want to add a default value (I think we shouldn't, it should be null), a new changeset should be created.

Reply:
added a changeset with some testvalues in #713

Reply:
default value should still be removed -- it also affects production databases